ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -712,7 +706,8 @@ - (void)respondToShoveGesture:(UIPanGestureRecognizer *)shoveRecognizer
 }
 #pragma mark Standard Initializer
 
-- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
+- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
+{
```

Comment:
I don't have any preference here, just tried to be consistent with initial styling, but we can switch based on what is more standard.